backend-api/template.yaml

```diff
@@ -773,6 +784,11 @@ Resources:
             Method: get
             RestApiId: !Ref SessionsApi
       Role: !GetAtt AsyncActiveSessionLambdaRole.Arn
+      Environment:
+        Variables:
+          KID_ARN: !GetAtt KMSEncryptionKey.Arn
+          STS_JWKS_ENDPOINT:
+            !FindInMap [EnvironmentVariables, !Ref Environment, STSJWKSENDPOINT]
```

Comment:
```suggestion
            !Sub https://mob-sts-mock.review-b-async.${Environment}.account.gov.uk/.well-known/jwks.json
```

Reply:
Will this work? Since this will be the value only in dev and build, prod will be a completely different URL

Reply:
Discussed with Sandy and agreed we do need mappings due to higher envs however this value will need to got dynamically when testing which will be added in future PR